test/unit/formidable.test.js

```diff
@@ -159,6 +160,92 @@ function makeHeader(filename) {
     form.emit('end');
   });
 
+  describe(`${name}#_onPart`, () => {
+    describe('when not allow empty files', () => {
+      describe('when file is empty', () => {
+        test('emits error when part is received', (done) => {
+          const form = getForm(name, {
+            multiples: true,
+            allowEmptyFiles: false,
+          });
+
+          const part = new Stream();
+          part.mime = 'text/plain';
+          // eslint-disable-next-line max-nested-callbacks
+          form.on('error', (error) => {
+            expect(error.message).toBe(
+              'options.allowEmptyFiles is false, file size should be greather than 0',
+            );
+            done();
+          });
+          form.onPart(part);
+          part.emit('end');
+          expect.assertions(1);
+        });
+      });
+
+      describe('when file is not empty', () => {
+        test('not emits error when part is received', () => {
+          const form = getForm(name, {
+            multiples: true,
+            allowEmptyFiles: false,
+          });
+          const formEmitSpy = jest.spyOn(form, 'emit');
+
+          const part = new Stream();
+          part.mime = 'text/plain';
+          form.onPart(part);
+          part.emit('data', Buffer.alloc(1));
+          expect(formEmitSpy).not.toBeCalledWith('error');
+        });
+      });
+    });
+
+    describe('when allow empty files', () => {
+      test('not emits error when part is received', () => {
+        const form = getForm(name, { multiples: true });
+        const formEmitSpy = jest.spyOn(form, 'emit');
+
+        const part = new Stream();
+        part.mime = 'text/plain';
+        form.onPart(part);
+        part.emit('end');
+        expect(formEmitSpy).not.toBeCalledWith('error');
+      });
+    });
+
+    describe('when file uploaded size is inferior than fileMinSize option', () => {
+      test('emits error when part is received', (done) => {
+        const form = getForm(name, { multiples: true, minFileSize: 5 });
+
+        const part = new Stream();
+        part.mime = 'text/plain';
+        form.on('error', (error) => {
+          expect(error.message).toBe(
+            'options.minFileSize (5 bytes) inferior, received 4 bytes of file data',
+          );
+          done();
+        });
+        form.onPart(part);
+        part.emit('data', Buffer.alloc(4));
+        expect.assertions(1);
+      });
+    });
+
+    describe('when file uploaded size is superior than fileMinSize option', () => {
```

Comment:
`minFileSize`

Reply:
Changed in [8c93a54](https://github.com/node-formidable/formidable/pull/662/commits/8c93a54c15817ad758466f877381edc2d1f1e669).